cmd/istioctl/istioctl_test.go

```diff
@@ -155,3 +155,19 @@ func TestNewGet(t *testing.T) {
 		t.Fatalf("Could not delete world-cb: %v", err)
 	}
 }
+
+func TestInvalidRouteRule(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-route-rule.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
+		t.Fatalf("Did not fail when presented with invalid route rule")
+	}
+}
+
+func TestInvalidDestinationPolicy(t *testing.T) {
+	rootSetup(t)
+	file = "testdata/invalid-destination-policy2.yaml"
+	if err := postCmd.RunE(postCmd, []string{}); err == nil {
```

Comment:
Then please rename the other test accordingly or the test data file accordingly.